src/common/exceptions/geo/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './geo.service.not.available.exception';
+export * from './geo.service.error.exception';
+export * from './geo.service.limit.reached.exception';
```

Comment:
```suggestion
export * from './geo.service.limit.exceeded.exception';
```

Also, rename the file.